tests/storages_tests/test_storages.py

```diff
@@ -1219,9 +1219,10 @@ def test_retry_failed_trial_callback(storage_mode: str, max_retry: Optional[int]
 
 
 @pytest.mark.parametrize("storage_mode", ["sqlite", "redis"])
-def test_fail_stale_trials(storage_mode: str) -> None:
+@pytest.mark.parametrize("grace_period", [None, 2])
+def test_fail_stale_trials(storage_mode: str, grace_period: int) -> None:
     heartbeat_interval = 1
-    grace_period = 2
+    _grace_period = 2
```

Comment:
I guess this code is a bit fragile since it depends on the value of `heartbeat_interval` implicitly (i.e., `grace_period = 2 * heartbeat_interval` if `grace_period` is `None`).
How about specifying the value explicitly?

Reply:
Ok, fixed.